src/components/Header/Header.js

```diff
@@ -102,23 +106,27 @@ class Header extends React.Component {
         patronDataReceived: patron.loggedIn || false,
         isFeatureFlagsActivated: {},
         logOutUrl: '',
+        featureFlagsStore: FeatureFlags.store.getState(),
```

Comment:
Why this update? If you're doing this you do not need to use `_extend` anymore.

Reply:
Before it is as this

```
this.state = _extend(
      {
        navData,
        loginCookieName: 'nyplIdentityPatron',
        loginCookieValue: null,
        patronName: patronNameObject.name || '',
        patronInitial: patronNameObject.initial || '',
        patronDataReceived: patron.loggedIn || false,
        isFeatureFlagsActivated: {},
        logOutUrl: '',
      },
      { featureFlagsStore: FeatureFlags.store.getState() },
    );
```

 I think there's not need to have a second object only for `featureFlagsStore`? And if this is the case we don't need `_extend` as well.

Reply:
Yeah, that's fine and yes, remove `_extend` if you're not going to use it. But, I think the reason those were separated is to make aware that `featureFlagsStore` is a completely different store.

Reply:
ohh if so then I can switch it back! I don't think it affect performance at all.

Reply:
Right, it wont'.